homeassistant/components/synologydsm/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "synologydsm",
   "name": "SynologyDSM",
   "documentation": "https://www.home-assistant.io/integrations/synologydsm",
-  "requirements": ["python-synology==0.3.0"],
+  "requirements": ["python-synology==0.4.0"],
```

Comment:
0.4.0 includes two things :
- "Added reverse compatibility with DSM 5.2" ProtoThis/python-synology#9
- "Make _get_url actually returning a response when retry_on_error=True" ProtoThis/python-synology#10 regarding "Fix SynologyDSM sensor unknown for 15 min" #30720

May you certify your Synology DSM sensors are up from HA start, and not after 15 min ? On DSM 5.2 I presume.
I'll test tonight/tomorrow if it's working on 6.2.

Will merge after that.

Reply:
Indeed, when testing in my local environment, I had the sensors showing up pretty much right after  HA started. And you're correct, tested with a DSM 5.2 NAS.

Reply:
Tested on DSM 6.2, works fine, from HA start 🎆